floors/enforce.go

```diff
@@ -0,0 +1,202 @@
+package floors
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/currency"
+	"github.com/prebid/prebid-server/exchange/entities"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// EnforceFloors does floors enforcement for bids from all bidders based on floors provided in request, account level floors config
+func EnforceFloors(bidRequestWrapper *openrtb_ext.RequestWrapper, seatBids map[openrtb_ext.BidderName]*entities.PbsOrtbSeatBid, account config.Account, conversions currency.Conversions) (map[openrtb_ext.BidderName]*entities.PbsOrtbSeatBid, []error, []*entities.PbsOrtbSeatBid) {
```

Comment:
Nitpick: we're in the floors package so you could probably just call this `Enforce`. The calling code will invoke it as `floors.Enforce`.

Reply:
done